File: 프로그래머스/1/12925. 문자열을 정수로 바꾸기/문자열을 정수로 바꾸기.py
def solution(s):
    # s = list(s)
    answer = 0
    negative = False
    
    if s[0] == "-":
        negative = True
        s = s[1:]
    elif s[0] == "+":
        s = s[1:]
    answer = int(s)
    if negative:
        answer = 0-answer
    
    return answer

File: 프로그래머스/1/12925. 문자열을 정수로 바꾸기/문자열을 정수로 바꾸기.cs
public class Solution {
    public int solution(string s) {
        bool N = false;
        int answer = 0;
        
        if(s[0] =='+')
        {
            answer = int.Parse(s.Substring(1));
        }
            
        else if(s[0] =='-')
        {
            answer = int.Parse(s.Substring(1));
            N = true;
        }
        else answer = int.Parse(s);
        
        if(N==true) answer = -answer;
            
        return answer;
    }
}